src/store/participantsStore.js

```diff
@@ -370,7 +370,7 @@ const mutations = {
 		const currentSpeakingState = state.speaking[token][sessionId].speaking
 
 		// when speaking has stopped, update the total talking time
-		if (!speaking && state.speaking[token][sessionId].lastTimestamp) {
+		if (currentSpeakingState && !speaking && state.speaking[token][sessionId].lastTimestamp) {
```

Comment:
should be fine, as this if() block intended to handle only situtations like `true` => `false`,